Source/CesiumRuntime/Public/CesiumMetadataModel.h

```diff
@@ -0,0 +1,61 @@
+
+#pragma once
+
+#include "CesiumFeatureTexture.h"
+#include "CesiumMetadataFeatureTable.h"
+
+#include "Containers/Array.h"
+#include "Containers/Map.h"
+#include "Containers/UnrealString.h"
+
+#include "CesiumMetadataModel.generated.h"
+
+namespace CesiumGltf {
+struct ExtensionModelExtFeatureMetadata;
+struct Model;
+} // namespace CesiumGltf
+
+USTRUCT(BlueprintType)
+struct CESIUMRUNTIME_API FCesiumMetadataModel {
+  GENERATED_USTRUCT_BODY()
+
+public:
+  FCesiumMetadataModel() {}
+
+  FCesiumMetadataModel(
+      const CesiumGltf::Model& model,
+      const CesiumGltf::ExtensionModelExtFeatureMetadata& metadata);
+
+private:
+  TMap<FString, FCesiumMetadataFeatureTable> _featureTables;
+  TMap<FString, FCesiumFeatureTexture> _featureTextures;
```

Comment:
@kring Following up from offline discussion, these are the only "duplicated" metadata that are being stored as part of the updates to the metadata blueprint api. Particularly the duplication is just the string keys; the mapped values are wrapped CesiumGltf views. In FCesiumMetadataFeatureTable a similar TMap already exists (from before this PR) with string keys for property names, but again the mapped property arrays are not being duplicated, FCesiumMetadataProperty still wraps a CesiumGltf view into property arrays just as before.